### PR TITLE
[9.0] extract project_task_department

### PR DESCRIPTION
**Pull request by yvaucher**
Move https://github.com/OCA/department/pull/55 here

Depends on 
- [x] https://github.com/OCA/project/pull/171

### CODE REVIEW

File: project_task_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# © 2011 Joël Grand-Guillaume (Camptocamp)
+# © 2013 Daniel Reis (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Project Task specific Department",
+    "version": "9.0.1.0.0",
+    "author": "Daniel Reis,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Generic Modules/Projects & Services",
+    "description": """\
```
Comment (yvaucher) [inline project_task_department/__openerp__.py:11]: To move in README.rst

Comment (dreispt) [inline project_task_department/__openerp__.py:11]: +1

File: project_task_department/models/project_task.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, fields
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+    department_id = fields.Many2one('hr.department', 'Department')
+
+    # Using old-style api for onchange
+    def onchange_project(self, cr, uid, ids, proj_id=False, context=None):
```
Comment (dreispt) [inline project_task_department/models/project_task.py:12]: Can you change this to the new API?

**Comment by dreispt** [comment]
Can you rebase please?

**Pull request closed without merging**